app/helpers/application_helper.rb

```diff
@@ -6,8 +6,4 @@ def url_checker(post)
   def time_and_date(post)
     post.created_at.strftime("%d/%m/%Y at %H:%M")
   end
-
-  def post_presenter(post)
-    simple_format("#{url_checker(post)} <br>Posted on #{time_and_date(post)}")
-  end
```

Comment:
I like this method. I'd keep using it and just remove the time/date bit from it.